pkgs/development/python-modules/polars/default.nix

```diff
@@ -49,7 +49,33 @@ buildPythonPackage {
   # ];
 
   meta = with lib; {
-    broken = (stdenv.isLinux && stdenv.isAarch64) || stdenv.isDarwin;
+    # Compiling arrow2 v0.10.1 (https://github.com/ritchie46/arrow2?branch=polars#da703ae3)
```

Comment:
this error message only happens when you apply the patch in https://github.com/NixOS/nixpkgs/pull/167959#issuecomment-1173160111

Reply:
I've amended the comment to include information about what's needed to get to the error.
Thank you for catching that and let me know if you still think it needs more polish!

Reply:
thank you! LGTM